client/client.go

```diff
@@ -23,39 +23,42 @@ func ConnectToServer(user, password, host string) (*ssh.Client, error) {
 	return client, nil
 }
 
-func Upload(client *ssh.Client, f string) {
+func Upload(client *ssh.Client, f string) error {
 	sftpClient, err := sftp.NewClient(client)
 	if err != nil {
 		log.Fatalf("Failed to create SFTP client: %v", err)
+		return err
```

Comment:
log.Fatalf exits the program completely. Cannot return after call to fatalf https://pkg.go.dev/log#Fatalf

baaki jagah use kiya hai usko bhi change karde